### ISSUE TITLE
Fold into main pouchdb library

### ISSUE THREAD

**Issue by jpike88** [pouchdb/upsert#63]
I noticed that pouchdb-find got folded into the main library, why not this one too? It's very useful

**Comment by indo-dev-0** [comment]
IMHO, it's a mistake that upsert wasn't included in CouchDB from the start!  CouchDB and friends are essentially denormalized datastores and are often used to serve data produced from more traditional relational stores.  At that point, you don't really care what the last version was, you just want to update it.

**Comment by erezsh** [comment]
I agree. Also, the code isn't maintained anymore, but the javascript world keeps on moving. It's better if it got folded into pouchdb for many reasons.
